src/test/java/com/mastercard/developer/interceptors/OkHttp2FieldLevelEncryptionInterceptorTest.java

```diff
@@ -4,7 +4,7 @@
 import com.mastercard.developer.encryption.EncryptionException;
 import com.mastercard.developer.encryption.FieldLevelEncryptionConfig;
 import com.mastercard.developer.test.TestUtils;
-import com.squareup.okhttp.*;
+import okhttp3.*;
```

Comment:
Best to avoid * imports

Reply:
Agree, but my IntelliJ keeps replacing it with okhttp3.*.